visma/solvers/simulEqn.py

```diff
@@ -0,0 +1,132 @@
+from visma.io.tokenize import tokenizer, getLHSandRHS
+from visma.simplify.simplify import simplifyEquation, moveRTokensToLTokens
+from visma.io.parser import tokensToString
+from visma.functions.constant import Constant
+from visma.functions.variable import Variable
+from visma.matrix.structure import SquareMat
+
+
+def coeffCalculator(LandR_tokens):
+    '''Finds coefficients of x, y and z and the constant term when solving for 3 simulaneous equations.
+
+    Arguments:
+        LandR_tokens -- 3 x 2 list -- each row contains left and right tokens of certain equation.
+
+    Returns:
+        coefficients -- 3 X 4 list -- each each row contains coefficients for x, y, z and constant term respectively.
+    '''
+    # TODO: Currently implemented taking with fixed variables x, y and z. To be implemented with different variables like u, v  and w.
+    coefficients = []
+    coefficients = [[0] * 4 for _ in range(3)]
+    for i, LandR_token in enumerate(LandR_tokens):
+        lTokens = LandR_token[0]
+        rTokens = LandR_token[1]
+        if len(rTokens) > 0:
+            lTokens, rTokens = moveRTokensToLTokens(
+                lTokens, rTokens
+            )
+        for _, token in enumerate(lTokens):
+            if isinstance(token, Variable):
+                if token.value == ['x']:
+                    coefficients[i][0] = token.coefficient
+                elif token.value == ['y']:
+                    coefficients[i][1] = token.coefficient
+                elif token.value == ['z']:
+                    coefficients[i][2] = token.coefficient
+            if isinstance(token, Constant):
+                coefficients[i][3] = token.value
+
+    return coefficients
+
+
+def crammerMatrices(coefficients):
```

Comment:
You can put the cramerMatrices in a new file (special.py) in visma/matrix and import it here.
Also, fix the typo Cramer in other places.

Reply:
done! :)